### PR TITLE
[one-cmds] Revise one-import-bcq as python script

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit revises one-import-bcq as python script.

Related : #4278
Draft: #4758
ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/one-cmds/one-import-bcq [hunk 1]
```diff
@@ -14,219 +19,206 @@
...
+import sys
+import tempfile
+
+import utils as _utils
+import generate_bcq_output_arrays as _bcq_info_gen
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description='command line tool to convert TensorFlow with BCQ to circle')
```
Comment (seanshpark) [inline compiler/one-cmds/one-import-bcq:34]: `command ` -> `Command`

Comment (mhs4670go) [reply compiler/one-cmds/one-import-bcq:34]: Actually, this is intended. `argparse` python library print default help message as small letter. So, I matched it.

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**